### DISCUSSION TITLE
Need help with Javascript

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [Panquesito7/Panquesito7#9] [Ideas]
Hi,

I added you as a collab for one of my projects, I need urgent help with JS, can you help me? Will give more details soon.

**Comment by Panquesito7** [comment]
Hey there! You can contact me via Discord to discuss this better: Panquesito7#3723.
I don't know anything about JavaScript, just a little bit, so I might not be able to help with that 😅 Thanks.

**Comment by KendallDoesCoding** [comment]
Will do that. Thanks

**Comment by KendallDoesCoding** [comment]
Please accept discord friend req.

**Reply by Panquesito7** [reply]
> Please accept discord friend req.

What is your username? Sorry, I have so many friend requests. 😅

**Reply by KendallDoesCoding** [reply]
> @Panquesito7 Second thought, I prefer talking on github discussions or email. Can I drop you a email?

**Reply by KendallDoesCoding** [reply]
Here's my discord username, I updated it for a bit, will change it back in sometime: spidermanisepic#9903

**Comment by KendallDoesCoding** [comment]
@Panquesito7 Second thought, I prefer talking on github discussions or email. Can I drop you a email?

**Reply by Panquesito7** [reply]
> @Panquesito7 Second thought, I prefer talking on github discussions or email. Can I drop you a email?

I would prefer Discord. I don't really use my mail to talk about stuff like this.

**Reply by KendallDoesCoding** [reply]
ok
